index.html

```diff
@@ -112,6 +112,14 @@ <h2>
       <li>Requesting a [=digital credential=], including mechanisms for secure
       presentation.
       </li>
+      <li>Ensuring that when an API call is made, the website does not learn
+      anything about the a holder's [=digital credentials=] or their software
+      unless the [=user agent=] has gained explicit user consent.
+      </li>
+      <li>Ensuring that any non-[=user agent=] software will not learn anything
```

Comment:
This one is little ambiguous.

Reply:
I was trying to generalize @leecam's notion of "wallet software". I'm wondering if we should call these things "native apps"? ... but then that leaves out "web apps", which could also hold/provide digital credentials. I think we do mean "any software that is not the user agent", but agree that the language is a bit unwieldy.

Reply:
I would avoid the term native as a prefix, based on comments of I seen on inclusive language elsewhere.

Platform specific is probably a better term.

Reply:
What about this?

```suggestion
      <li>Ensuring that any platform-specific software will not learn anything
```

OR

```suggestion
      <li>Ensuring that any installed application software will not learn anything
```

/cc @samuelgoto

Reply:
Hmm, what about "software that is not involved in the consent-seeking process"? (though that's an unfortunate mouthful).

What we're trying to say here is that "The request is supposed to start off super secret and is only to be exposed to the holder via the software that they use to signal consent (the browser). After that point, the software that they chose to receive the request, and only that piece of software, is supposed to receive the request."